projects/gui/src/newtournamentdialog.cpp

```diff
@@ -192,6 +195,17 @@ bool NewTournamentDialog::canStart() const
 	if (m_addedEnginesManager->engineCount() < 2)
 		return false;
 
+	QPushButton* button = ui->buttonBox->button(QDialogButtonBox::Ok);
+
+	// check for duplicate configuration names
+	if (m_addedEnginesManager->engineNames().count()
+	!=  m_addedEnginesManager->engineCount())
+	{
+		button->setText("Resolve Duplicates!");
```

Comment:
Can you wrap the "Resolve Duplicates!" text inside `tr()` so that it can be localized?

Reply:
ah, good catch!